### ISSUE TITLE
Multiple CancellationTokens as the last parameters of a function trigger CA1068

### ISSUE THREAD

**Issue by Cyberboss** [dotnet/roslyn-analyzers#1816]
#### Analyzer package

[Microsoft.CodeAnalysis.FxCopAnalyzers](https://www.nuget.org/packages/Microsoft.CodeAnalysis.FxCopAnalyzers)

#### Package Version

v2.6.1

#### Diagnostic ID

CA1068: Method Tgstation.Server.Host.Core.IJobManager.WaitForJobCompletion(Tgstation.Server.Host.Models.Job, Tgstation.Server.Host.Models.User, System.Threading.CancellationToken, System.Threading.CancellationToken)' should take CancellationToken as the last parameter

#### Repro steps

Function definition:
```c#
Task WaitForJobCompletion(Job job, User canceller, CancellationToken jobCancellationToken, CancellationToken cancellationToken);
```

#### Expected behavior

The warning should not appear since there is a `CancellationToken` as the last parameter and each token has a different meaning

#### Actual behavior

The warning appears

**Comment by davkean** [comment]
Have the same situation:

``` C#
private bool TryTranslateException(Exception exception, CancellationToken cancellationToken1, CancellationToken cancellationToken2)
```
